interfaces/ledger/ledgerInterface.h

```diff
@@ -94,32 +94,32 @@ class LedgerInterface
      * @param _txHash hash of transaction
      * @param _onGetTx
      */
-    virtual void asyncGetTransactionReceiptByHash(
-        h256 const& _txHash, std::function<void(Error::Ptr, bcos::protocol::TransactionReceipt::ConstPtr)> _onGetTx) = 0;
+    virtual void asyncGetTransactionReceiptByHash(h256 const& _txHash,
```

Comment:
Please modify `h256` to `HashType`, Similar modifications are required below.

Reply:
done